### DISCUSSION TITLE
As a teacher, I can't apply for education benefits using my faculty email address.

### DISCUSSION THREAD

**Discussion by LightingMax** [community/community#53935] [GitHub Education]
### Hi everyone,

I am a teacher from Chongqing University of Arts and Sciences in China. I had a problem using my faculty email address when applying for education benefits.
My faculty email address is 20010004@cqwu.edu.cn which ends with -cqwu.edu.cn
. But Github the official website requires me to use the -stu.cqwu.edu.cn  suffix email.
This is out of line with our actual situation.
I wonder if teachers at other colleges have similar problems to mine.
Hopefully, GitHub officials will be able to see and fix our problems.

**Comment by dcecho** [comment]
You've entered the wrong channel. ”stu“ is the email suffix for verifying student identity。Sorry， it took me so long to see this question.
